.github/workflows/trigger-docker-image-build.yml

```diff
@@ -0,0 +1,47 @@
+name: Trigger Docker image build
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - current
+
+permissions:
+  packages: write
+  contents: read
+  attestations: write
+  id-token: write
+  pull-requests: read
+
+jobs:
+  track-changes:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    
+    env:
+      REF: main  # Used for curl to trigger image build

```

Comment:
We do not have a `main` branch - we use `current`

Reply:
@c-po We use this `main` branch in our repository where we store reusable workflows and there is no plan to use `current`, `circinus` or `sagitta` branches there. Also all our previous PRs referred to this `main` branch.